lib/loggly/common.js

```diff
@@ -147,7 +147,7 @@ common.loggly = function () {
     }
   }
   var requestOptions = {
-    uri: isBulk ? uri + '/tag/' + headers['X-LOGGLY-TAG'] : uri,
+    uri: (isBulk && headers['X-LOGGLY-TAG']) ? uri + '/tag/' + headers['X-LOGGLY-TAG'] : uri,
```

Comment:
Add **headers['X-LOGGLY-TAG']** property into the request header only when the logging mode is Bulk and **headers['X-LOGGLY-TAG']** contains some value. This will prevent logs sending from the **undefined** tag name.